src/api/models/index.js

```diff
@@ -8,7 +8,9 @@ const sequelize = new Sequelize(env.dbUrl, {
 
 const models = {
   User: sequelize.import('./user'),
-  Token: sequelize.import('./token')
+  Token: sequelize.import('./token'),
```

Comment:
@truestbyheart rebase with develop. We no longer need `Token` and `SequelizeMeta` tables.